src/Stryker.Core/README.md

```diff
@@ -39,6 +40,19 @@ Stryker supports a variety of mutators, which are listed below. Do you have a su
 | `true` | `false` |
 | `false` | `true` |
 
+## Assignment mutator
+| Original | Mutated | 
+| ------------- | ------------- | 
+|`+= `	| `-= ` |
+|`-= `	| `+= ` |
+|`*= `	| `/= ` |
+|`/= `	| `*= ` |
+|`%= `	| `*= ` |
+|`<<=`  | `>>=` |
+|`>>=`  | `<<=` |
+|`&= `	| `\|= ` |
```

Comment:
The mutator uses `SyntaxKind.ExclusiveOrAssignmentExpression` which is `^=`.
`|=` is `SyntaxKind.OrAssignmentExpression`.